changelogs/unreleased/gh-8497-promote-atomic.md

```diff
@@ -1,4 +1,4 @@
 ## bugfix/replication
 
-* Fixed possible failure to promote the desired node by `box.ctl.promote()` on
+* Fixed a possible failure to promote the desired node by `box.ctl.promote()` on
```

Comment:
This change better be in the previous commit.

Reply:
Thanks for noticing! Fixed.